src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,105 @@
+import { Bookmark, User, Article } from '../sequelize/models';
+
+/**
+ * @description holds bookmarks logic
+ */
+class BookmarkController {
+  /**
+   * @description create bookmarks
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} returns object with created bookmark
+   */
+  static async createBookmark(req, res) {
+    try {
+      const userId = req.userData.id;
+      const articleSlug = req.params.slug;
+      const user = await User.findOne({ where: { id: userId } });
+      if (!user) {
+        return res.status(403).json({
+          error: 'Provided token is not registered to you'
```

Comment:
What did you try to mean here?

Reply:
This means the user id supplied in the token is not in our DB.